src/agb_eeprom.c

```diff
@@ -0,0 +1,273 @@
+#include "global.h"
+#include "gba/eeprom.h"
+
+
```

Comment:
(I can't leave a comment in data1.s since it's too large, so I'm putting the comment here)
```c
/*
I don't know how to rename or abstract above data(gUnk_08B6CF4C), witch are nothing but zeros in the rom. 
*/
```
Two mistakes: 
abstract -> extract
witch -> which

And if you're sure this is never referenced (that means, the name never appears in any other parts of the repo and there's no pointer to it when you search in a hex editor), you can decompile it to anything you want, like `static const u8 sUnused[0x100] = {}`;